### ISSUE TITLE
when smtp enabled = true, smtp port error

### ISSUE THREAD

**Issue by kulacino** [getlago/lago-helm-charts#10]
Here is what I get when I tried to enable smtp. I filled out the detail such as address, username, fromEmail, etc. But an error was shown in port.

Here is the value:

```
smtp:
    enabled: true
    address: smtp.mydomain.com
    port: 587
    username: tools@mydomain.com
    password: mySMTPpassword
    fromEmail: noreply@mydomain.com
```

Screenshot attached. 
![error-lago](https://github.com/getlago/lago-helm-charts/assets/9045794/49f3b5cf-f156-4cca-bd2c-86c6d5e22af1)

**Comment by kulacino** [comment]
Hi. I think you're missing `.Values.global.smtp.port` in the worker-deployment and api-deployment.

**Comment by jdenquin** [comment]
fixed in #11
